### PR TITLE
feat: English & French Language Support

### PR DESCRIPTION
**Pull request by dhruvilp**
**Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the PR).**
- [ ] REQUIRED: Do you have an Issue **assigned to you** within the v1 milestone for this PR?  Put the Issue number here: #640 
- [x] Provided detailed pull request description and a succinct title (consider template below for guidance).
- [x] Tested your changes, especially after any code review iterations.
- [x] Included any relevant screenshots of UI updates.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [x] Verified all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE) file in the root of the repository.
- [x] Verified your name is in the [content/credits.yaml](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) file (if you want it to be).

After all boxes above are checked, request and receive an Approved review from any team member knowledgable in the area (TODO team member list).  Once approved, the team member will assign your review to a Committer or use the `needs-merge` label.

## What does this PR accomplish?

"Official 6 Language Support"

Implemented Settings Page UI based on Sigma Designs + Added selectable buttons to switch between official 6 languages.

Addressed: 

## Did you add any dependencies?

No

## How did you test the change?

On my device (Pixel 3a XL)

## Screenshots of new UI
Spanish Lang Support
Arabic Lang Support

### CODE REVIEW

**Comment by hspinks** [comment]
@dhruvilp thanks for this contribution! can you please separate this into 2 PRs separating the concerns between language selection and the opt-out toggle? as we consider which features are "must"s, this will help us in case we need to choose. the first priority should just be the opt-out toggle.

**Comment by dhruvilp** [comment]
> @dhruvilp thanks for this contribution! can you please separate this into 2 PRs separating the concerns between language selection and the opt-out toggle? as we consider which features are "must"s, this will help us in case we need to choose. the first priority should just be the opt-out toggle.

Sure thing, @hspinks!

**Comment by dhruvilp** [comment]
@hspinks Did it! Checkout #658 PR for opt-out-toggle feature

**Comment by advayDev1** [comment]
@dhruvilp can this just be english and french? and can you sync to HEAD?

**Comment by dhruvilp** [comment]
> @dhruvilp can this just be english and french? and can you sync to HEAD?

Sure thing!

**Comment by dhruvilp** [comment]
> @dhruvilp can this just be english and french? and can you sync to HEAD?

@advayDev1 Just pushed updates!

**Comment by dhruvilp** [comment]
> @dhruvilp can this just be english and french? and can you sync to HEAD?

@advayDev1 @hspinks All checks have passed, good to go!

File: content/credits.yaml [hunk 1]
```diff
@@ -48,9 +48,13 @@ team:
   - Chris Hosmer
   - Vivian Cromwell
   - Shilpa Apte
+<<<<<<< HEAD
+
+=======
```
Comment (britannio) [inline content/credits.yaml]:
```suggestion
```

File: content/credits.yaml [hunk 2]
```diff
@@ -48,9 +48,13 @@ team:
...
   - Ursula Zhao
   - Evan Pye
   - Moosphon
+>>>>>>> a08d340b83c4933ae4bc7aadeeed66564b382b28
```
Comment (britannio) [inline content/credits.yaml]:
```suggestion
```

File: content/credits.yaml (hunk 1 above)
Comment (britannio) [reply content/credits.yaml]: should this be here?

File: client/flutter/lib/pages/settings_page.dart [hunk 3]
```diff
@@ -74,9 +68,48 @@ class _SettingsPageState extends State<SettingsPage> {
                   style: Theme.of(context).textTheme.subhead,
                 ),
               ),
-
-              /// TODO: Implement UI:-
-              /// TODO:   selection of language preferences already created PR for it (#654)
+              Padding(
+                padding: const EdgeInsets.all(15.0),
+                child: SelectableText(
```
Comment (britannio) [inline client/flutter/lib/pages/settings_page.dart]: Why `SelectableText` instead of `Text`?

Comment (dhruvilp) [reply client/flutter/lib/pages/settings_page.dart]: So, it can be selected (just for good practice) when we deploy to web in the future.

Comment (dhruvilp) [reply client/flutter/lib/pages/settings_page.dart]: @britannio Sorry for the merge conflicts texts.

Comment (britannio) [reply client/flutter/lib/pages/settings_page.dart]: I don't know of plans to target web in the future, maybe @advayDev1 can confirm?

Comment (britannio) [reply client/flutter/lib/pages/settings_page.dart]: Screenshot_20200331-212858__01
Also, the constrast is wrong on my device.

Comment (dhruvilp) [reply client/flutter/lib/pages/settings_page.dart]: The reason behind that bug isn't your device, it's `accentColor` in app theme. I think accentColor should not be closer to `white` if our default theme is `white`. Flutter has separate themes for fonts `textTheme` so, I think we've to use that. Until then, I could just replace `SelectableText` with `Text` widgets for now. @britannio

Comment (britannio) [reply client/flutter/lib/pages/settings_page.dart]: Using a `Text` widget solves the issue

Comment (dhruvilp) [reply client/flutter/lib/pages/settings_page.dart]: Well, yeah but I strongly suggest to use built-in TextTheme for styling texts in the app.

**Comment by dhruvilp** [comment]
Note: Do not merge this YET! I've fix coming up soon. Shared Pref is missing currently so I'm adding it.

**Comment by dhruvilp** [comment]
@hspinks @advayDev1 This PR is good to go! I tested thoroughly so this could be merge anytime you can.

**Comment by hspinks** [comment]
@dhruvilp thanks for this contribution, but since we will not be launching with multi-language support for v1, I'm going to close this PR for now. We can re-examine when we add support for more languages

**Comment by dhruvilp** [comment]
@hspinks No problem at all 👍

**Pull request closed without merging**